packages/components/src/mobile/bottom-sheet/range-cell.native.js

```diff
@@ -212,7 +219,7 @@ class BottomSheetRangeCell extends Component {
 				] }
 				cellRowContainerStyle={ containerStyle }
 				accessibilityRole={ 'none' }
-				value={ '' }
+				leftAlign
```

Comment:
Are we sure we don't need `value`? I'd also check that setting `leftAlign` doesn't affect other places where this is used.

Reply:
@geriux for additional context, [I left a note in the commit making this change](https://github.com/WordPress/gutenberg/pull/25810/commits/17ae30be9ab6825a791d6acd0859f5fe9e924bcb). After reading [some of the history of the `value` prop](https://github.com/WordPress/gutenberg/pull/19039#discussion_r356463479) and testing various areas of the application, no, I do not believe we need this value. It's inclusion appeared to be solely related to text alignment, but it was causing rendering issues in the `RangeCell`s where I needed to re-mount to sync coordinates to them.

I will definitely make another pass through the application UI to see if I notice any regressions in styling, though. Let me know what you think about the history I found.

Reply:
Thank you for explaining a bit more about it! I agree with my comment from back then =D

Let's keep this. 👍